package excepciones;

public class TestException2 {

	public static void main(String[] args) {
		
	    int x=10, y=0;
		double division;
		
		try {
			// C칩digo que se intentar치 ejecutar...
			division=x/y;
			System.out.printf("\n\t La divis칩n es: %.3f",division);
		}catch(ArithmeticException exception) {
			System.out.print("\n\t Error, no puedes dividir por cero.");
			System.out.print("\n\t Error: "+exception.getMessage());
		}
		
		System.out.println("\n\t Excepci칩n controlada...");
		
		
	}

}
